### PR TITLE
Android release

### PR DESCRIPTION
**Pull request by chungvl**
Also fixed "view current location button" issue on map for iOS (positioned at the bottom right, which interferes with the FABs)

### CODE REVIEW

File: createAndroidBuild.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ionic cordova build --release android
+~/Android/Sdk/build-tools/27.0.3/zipalign -v -p 4 platforms/android/app/build/outputs/apk/release/app-release-unsigned.apk $$.1.apk
+~/Android/Sdk/build-tools/25.0.0/apksigner sign --ks my-release-key.jks --out `date +"%Y%m%d-%H%M%S"`.apk $$.1.apk
```
Comment (tony-gaia) [inline createAndroidBuild.sh]: Can the key file be called something more descriptive like "koala-watch-release-key". Also, can you put it in rattic?

File: src/components/records-list/records-list.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Component, Input } from '@angular/core';
 import { NavController, NavParams } from 'ionic-angular';
 import { ClientRecord } from '../../shared/interfaces/mobile.interfaces';
 import { ANY_ANGULAR_DATETIME_FORMAT } from '../../biosys-core/utils/consts';
-import { RECORD_BLUE, RECORD_GREEN, RECORD_RED } from '../../shared/utils/consts'
+import { RECORD_BLUE, RECORD_GREEN, RECORD_RED } from '../../shared/utils/consts';
```
Comment (tony-gaia) [inline src/components/records-list/records-list.ts]: I've changed these to RECORD_INCOMPLETE, RECORD_COMPLETE and RECORD_UPLOADED.

Just saying the colour is not quite ideal (as, for instance, the red isn't actually red) - rather the name should indicate what it represents.

File: src/environments/environment.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 export const environment = {
     production: false,
-    server: 'http://localhost:8000',
+    server: 'https://staging-biosys.gaiaresources.com.au',
```
Comment (tony-gaia) [inline src/environments/environment.ts]: Don't commit this change

File: src/pages/home/home.html [hunk 4]
```diff
@@ -16,18 +16,22 @@
...
-    <ion-fab bottom right style="right: 80px">
-        <button ion-fab (click)="onClickedNewRecord('Koala Opportunistic Observation')">
-            <img alt="Koala Opportunistic Observation" class="home-fab-icon" src="assets/imgs/white_eye.png">
-        </button>
-    </ion-fab>
-    <ion-fab bottom right>
-        <button ion-fab (click)="onClickedNewRecord('KLM-SAT Census')">
-            <img alt="KLM-SAT Census" class="home-fab-icon" src="assets/imgs/white_poop.png">
-        </button>
+        <ion-row>
```
Comment (tony-gaia) [inline src/pages/home/home.html]: How come the buttons are all on the left now?

Comment (chungvl) [reply src/pages/home/home.html]: The iOS Google Map places the "go to my location" button at the bottom right corner (different to Android, which is the top Right corner).

Without this change the FABs overlap the "go to my location" button.

File: createAndroidBuild.sh (hunk 1 above)
Comment (chungvl) [reply createAndroidBuild.sh]: Yep, done.

**Pull request merged**